app/components/gcse_row_content_component.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if course.gcse_section_complete? %>
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= required_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= pending_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= gcse_equivalency_content(course) %>
+  </p>
+
+  <% if course.additional_gcse_equivalencies %>
+    <p class="govuk-summary-list-value">
+      <%= course.additional_gcse_equivalencies %>
+    </p>
+  <% end %>
+<% else %>
+  <%= govuk_inset_text(classes: %w[app-inset-text app-inset-text--important]) do
+        tag.p("GCSEs and equivalency tests", class: "govuk-heading-s app-inset-text__title") +
+          govuk_link_to(
+            "Enter GCSEs and equivalency test requirements",
```

Comment:
I think we might need to tweak this slightly:

```suggestion
            "Enter GCSE and equivalency test requirements",
```

What do you think @B-Bonner?